front/app/modules/commercial/insights/admin/containers/Reports/charts/IdeasByStatusChart.tsx

```diff
@@ -125,19 +108,35 @@ export class IdeasByStatusChart extends React.PureComponent<
             </NoDataContainer>
           ) : (
             <ResponsiveContainer
-              height={serie.length > 1 ? serie.length * 50 : 100}
+              height={
+                sortedByValue.length > 1 ? sortedByValue.length * 50 : 100
+              }
             >
-              <BarChart data={serie} layout="vertical" ref={this.currentChart}>
+              <BarChart
+                data={sortedByValue}
+                layout="vertical"
+                ref={this.currentChart}
+                margin={{
```

Comment:
This one, as well as barSize, might make a good shared const or context for consistency ?

Reply:
Good point! Re `barSize`, I've added it to the chart theme to be reused from theme :) 
On the margins - there is a bit more variance between the charts and it might also change in the future so I would leave it flexible for now but keep an eye on it and maybe abstract it in the future.